ultron-compose/src/androidMain/kotlin/com/atiurin/ultron/core/compose/view/ViewInComposeExt.kt

```diff
@@ -0,0 +1,41 @@
+package com.atiurin.ultron.core.compose.view
+
+import android.view.View
+import androidx.compose.ui.InternalComposeUiApi
+import androidx.compose.ui.node.InteroperableComposeUiNode
+import androidx.compose.ui.test.SemanticsNodeInteraction
+import com.atiurin.ultron.core.compose.nodeinteraction.UltronComposeSemanticsNodeInteraction
+import org.hamcrest.Description
+import org.hamcrest.Matcher
+import org.hamcrest.TypeSafeMatcher
+
+/**
+ * Extract view from [androidx.compose.ui.viewinterop.AndroidView] in Compose
+ */
+fun UltronComposeSemanticsNodeInteraction.fetchView(): View? = semanticsNodeInteraction.fetchView()
+
+/**
+ * Extract view from [androidx.compose.ui.viewinterop.AndroidView] in Compose
+ */
+@OptIn(InternalComposeUiApi::class)
```

Comment:
We don't interact with SemanticsNodeInteraction directly, so this method should be deleted